src/test/kotlin/coverosR3z/timerecording/UITimeEntryTests.kt

```diff
@@ -138,11 +140,34 @@ class UITimeEntryTests {
     @Test
     fun `timeentry - I should see my existing time entries when I open the time entry page`() {
         val s = TimeEntryUserStory.getScenario("timeentry - I should see my existing time entries when I open the time entry page")
+
+        addSomeTimeEntries()
         s.markDone("Given I had previous entries this period")
+
+        verifyTimeEntries()
         s.markDone("when I open the time entry page")
         s.markDone("then I see my prior entries")
     }
 
+    private fun verifyTimeEntries() {
+        // Verify the entries
+        pom.driver.get("${pom.domain}/${ViewTimeAPI.path}")
+
+        assertEquals("1.00", pom.vtp.getTimeForEntry(1))
+        assertEquals("2021-01-01",  pom.vtp.getDateForEntry(1))
```

Comment:
Any chance this assertion will fail in Chrome vs. Other due to date formatting?

Reply:
Hmm good point!

Reply:
Checked - it did work on Firefox as well.